cli/board/list.go

```diff
@@ -100,15 +101,17 @@ func outputListResp(ports []*rpc.DetectedPort) {
 			for _, b := range boards {
 				board := b.GetName()
 				fqbn := b.GetFQBN()
-				t.AddRow(address, protocol, board, fqbn)
+				core := fqbn[:strings.LastIndex(fqbn, ":")]
```

Comment:
we have some infra for FQBN parsing here https://github.com/arduino/arduino-cli/blob/master/arduino/cores/fqbn.go#L36 that would be nice using here.